ui/app/src/utils/path.ts

```diff
@@ -205,3 +205,26 @@ export function getCodeEditorSrc({
   });
   return `${authoringBase}/legacy/form${qs}`;
 }
+
+export function expandPathMacros(path: string) {
```

Comment:
Let's go with the `values` argument that would internally call another `format` util.